### PR TITLE
Changes needed to successfully integrate with Skillz SDK

### PR DESCRIPTION
**Pull request by millibeckers**
Tested by building locally inside the SDK

### CODE REVIEW

**Review by PaigeKH — APPROVED** [review]
🐉

File: mopub-sdk/mopub-sdk-base/release.gradle [hunk 1]
```diff
@@ -0,0 +1,104 @@
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'signing'
+
+repositories {
+    maven { url 'http://nexus.skillz.com/content/repositories/thirdparty/' }
+}
+
+def configureReactNativePom(def pom) {
```
Comment (TGier) [inline mopub-sdk/mopub-sdk-base/release.gradle]: This name is a tad misleading.

File: mopub-sdk/mopub-sdk-base/release.gradle [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    uploadArchives {
+        configuration = configurations.archives
+
+        repositories.mavenDeployer {
+            beforeDeployment {
+                MavenDeployment deployment -> signing.signPom(deployment)
+            }
+
+            repository(url: 'http://nexus.skillz.com/content/repositories/thirdparty/') {
+                authentication(userName: "garbage", password: "garbage")
```
Comment (TGier) [inline mopub-sdk/mopub-sdk-base/release.gradle:86]: 🗑️

File: mopub-sdk/mopub-sdk-base/release.gradle (hunk 1 above)
Comment (millibeckers) [reply mopub-sdk/mopub-sdk-base/release.gradle]: yeah i just copied and pasted this from one of our other ones, ill fix it though

**Review by TGier — APPROVED** [review]
👍

**Pull request merged**